picard/file.py

```diff
@@ -709,12 +709,12 @@ def update(self, signal=True):
                         self.state = File.CHANGED
                     break
             else:
-                if (self.metadata.images
+                self.similarity = 1.0
+                if self.state in (File.CHANGED, File.NORMAL):
```

Comment:
Just a minor thing, but using a set instead of a tuple  seems more correct to me:

```suggestion
                if self.state in {File.CHANGED, File.NORMAL}:
```

Reply:
Definitely, yes, I changed it